python/sglang/srt/layers/quantization/modelopt_quant.py

```diff
@@ -481,6 +481,77 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
                 layer.w2_input_scale.max(), requires_grad=False
             )
 
+        # Align FP8 weights to FlashInfer per-tensor kernel layout if enabled
+        if should_use_flashinfer_trtllm_moe():
+            from flashinfer import reorder_rows_for_gated_act_gemm, shuffle_matrix_a
+
+            # 1) Swap W13 halves: [Up, Gate] -> [Gate, Up] expected by FI
```

Comment:
can we make sure this is not a duplicate step from https://github.com/sgl-project/sglang/blob/7e6191c098e9ff610f9f9f20376bad9071908dc9/python/sglang/srt/layers/moe/fused_moe_triton/layer.py#L609?

It seems we already do the swap during the weight loading instead of the process_weights_after_loading.